doc/source/api/diffpy.fourigui.rst

```diff
@@ -14,17 +12,12 @@ Subpackages
 -----------
 
 .. toctree::
-   diffpy.fourigui.example_package
-
-Submodules
-----------
-
-|module|
---------
-
-.. |module| replace:: diffpy.fourigui.example_submodule module
-
-.. automodule:: diffpy.fourigui.example_submodule
-    :members:
-    :undoc-members:
-    :show-inheritance:
+    :titlesonly:
+
+    diffpy.fourigui.dist
```

Comment:
Please check this.  I doubt we want most of these in the documentation.  Also, let's check how they got here....was it a problem with the API process that is also affecting other packages or something that went wrong locally?

Reply:
@sbillinge The API section of the cookiecutter generates documentation for all files (see screenshot, not all files are listed). Not quite sure how to handle this. I will discuss with others 
![Screenshot 2024-10-08 at 11 00 18 AM](https://github.com/user-attachments/assets/00a0a589-07ba-4701-84a8-28ac497eaa6d)

Reply:
Just to double check:

Did you run the following? 

```
python ../../release-scripts/auto_api.py diffpy.fourigui ./src/diffpy/fourigui ./doc/source/api
```

Reply:
I am afraid it is due to the relative path issue. If you have the same problem, let's figure this out.